src/Microsoft.Extensions.Caching.Redis/RedisCache.cs

```diff
@@ -34,14 +33,14 @@ public class RedisCache : IDistributedCache, IDisposable
         private readonly RedisCacheOptions _options;
         private readonly string _instance;
 
-        public RedisCache(IOptions<RedisCacheOptions> optionsAccessor)
+        public RedisCache(RedisCacheOptions options)
         {
-            if (optionsAccessor == null)
+            if (options == null)
             {
-                throw new ArgumentNullException(nameof(optionsAccessor));
+                throw new ArgumentNullException(nameof(options));
             }
 
-            _options = optionsAccessor.Value;
+            _options = options;
 
```

Comment:
Hmm, the redis constructor doesn't validate any of the required arguments like the sql one does. This makes the "You failed to specify RedisCacheOptions" mistake harder to detect.